dashboard/app/models/levels/blockly.rb

```diff
@@ -379,6 +380,33 @@ def localized_blockly_level_options(script)
     options.freeze
   end
 
+  def localize_blockly_level_options_for_lab2(script)
+    options = Rails.cache.fetch("#{cache_key}/#{script.try(:cache_key)}/#{I18n.locale}/localized_blockly_level_options", force: !Unit.should_cache?) do
```

Comment:
Not super familiar with how the caching works, but should this be using a different/specific cache key? Looks like this is the same cache key used in `localized_blockly_level_options`?

Reply:
Thanks @sanchitmalhotra126 and @Hamms for both catching this.